test/spec/helper.state.js

```diff
@@ -361,7 +361,7 @@ test('getConfigurationFromQueryString should parse page as number and be consist
 
   var queryString = helper.getStateAsQueryString({filters: filters});
 
-  var partialStateFromQueryString = algoliasearchHelper.AlgoliaSearchHelper.getConfigurationFromQueryString(
```

Comment:
Do we have a check that ensure that the previous `algoliasearchHelper.AlgoliaSearchHelper.` methods are still there and works?